.env

```diff
@@ -1,6 +1,6 @@
 COMPOSE_PROJECT_NAME=litt
-REMOTE_HOST=litsiden-p01.dbc.dk
-REMOTE_DB_DUMP_CMD='/data/www/litteratursiden_dk/htdocs/vendor/bin/drush --root=/data/www/litteratursiden_dk/htdocs/web --uri=litteratursiden.dk sql-dump'
+REMOTE_HOST=litteratursiden.dk
```

Comment:
Not sure we should change from `REMOTE_HOST=litsiden-p01.dbc.dk`, so as not to break ssh remote host identification...

Reply:
We assume that the ssh user is set in `~/.ssh/config` and then we can just as well add 

```
Host litteratursiden.dk
  HostName litsiden-p01.dbc.dk
  User deploy
  ForwardAgent yes
``` 

Doing it like this, we don't have to use the real server name in the code (in this repository).

Reply:
The `REMOTE_DB_DUMP_CMD` should not be removed

Reply:
No, and I don't remove it (it's in line 3; cf. https://github.com/litteratursiden/litteratursiden/pull/1/files/1492515c2e2d4d86a078ed2915e938d3b221cab0#diff-f579cccc964135c7d644c7b2d3b0d3ecR3).